cmd/mattermost/commands/server_test.go

```diff
@@ -105,9 +105,9 @@ func TestRunServerSystemdNotification(t *testing.T) {
 	socketReader := make(chan string)
 	go func(ch chan string) {
 		buffer := make([]byte, 512)
-		count, err := connection.Read(buffer)
-		if err != nil {
-			panic(err)
+		count, readErr := connection.Read(buffer)
```

Comment:
I don't have an objection to the naming change to avoid shadowing (as previously discussed), but I want to call that out that I don't think `it's needed here` is true in the sense of correctness. 

The shadowing `err` declared here is a new variable which won't conflict with the shadowed `err` in the outer scope. The shadowing warning is a reminder that you can no longer access the value in the outer scope, but we don't actually want to do that here anyway, so it's all "correct".

👍 for driving the removing of shadowing anyway.

Reply:
Okay, I see. You'r right.